src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md

```diff
@@ -4,14 +4,38 @@ description: Links to Pupil Invisible Monitor app.
 
 # Monitor your Data Collection in Real-Time
 
+Using the **Pupil Invisible Monitor** app you can easily monitor you data collection in real-time and remote control all your Pupil Invisible glasses from another device. It's a web-app that can be opened in any browser on a computer, tablet or phone! The only requirement is that the Pupil Invisible Companion device and the device you use to access the Monitor app are connected to the same network.
+
+To access the Monitor app make sure the Pupil Invisible Companion app is running and visit the page [pi.local](http://pi.local) on your monitoring device.
+
+
+## The App's User Interface
+Once the page has loaded, you will be presented with a live stream of the scene camera with a gaze overlay. On the right-hand side you can find a record button, that allows you to remotely start and stop a recording on the device.
+
+If you have multiple Pupil Invisible devices connected to the network, you can switch between devices using the switch button.
+
+Using the various event buttons, you can remotely save an [event](/invisible/explainers/basic-concepts/#events) in a recording to mark points of interest. In the settings view you can customize the according event names.
+
 <div style="display:flex;justify-content:center;" class="pb-4">
   <v-img
-    :src="require('../../../media/invisible/how-tos/pi-monitor-app.jpg')"
-    max-width=80%
+    :src="require('../../../media/invisible/how-tos/monitor-ui.png')"
+    max-width=100%
   >
   </v-img>
 </div>
 
-**Coming soon!** We are finishing up work on a new version of Pupil Invisible Monitor, which will allow you to easily monitor and remote-control all Pupil Invisible devices in your local network from your browser.
+## Connection Problems?
+
+In case you have trouble connecting to your Pupil Invisible device please consider the following:
+- First, make sure the Companion phone and the device you are using for monitoring are both connected to the same network.
+- The the local network must allow MDNS and UDP traffic for the app to work. In large public networks this may be prohibited for security reasons. You can circumvent this by running a separate WiFi using the phone's hot spot functionality or an extra router.
```

Comment:
Links to MDNS and UDP?

Reply:
Where would those point to? Wikipedia? Not sure if that would add much value but, wouldn't hurt either!

Reply:
I think its ok without. The solution is listed in the text. Knowing more about MDNS will not help the avg. user.